src/config/roles.php

```diff
@@ -112,12 +112,12 @@
     'rolesGuiMiddleware'            => env('ROLES_GUI_MIDDLEWARE', 'role:admin'),
 
     // User Permissions or Role needed to create a new role
-    'rolesGuiCreateNewRolesMiddlwareType'   => env('ROLES_GUI_CREATE_ROLE_MIDDLWARE_TYPE', 'role'), //permissions or roles
-    'rolesGuiCreateNewRolesMiddlware'       => env('ROLES_GUI_CREATE_ROLE_MIDDLWARE_TYPE', 'admin'), // admin, XXX. ... or perms.XXX
+    'rolesGuiCreateNewRolesMiddlewareType'   => env('ROLES_GUI_CREATE_ROLE_MIDDLEWARE_TYPE', 'role'), //permissions or roles
+    'rolesGuiCreateNewRolesMiddleware'       => env('ROLES_GUI_CREATE_ROLE_MIDDLEWARE_TYPE', 'admin'), // admin, XXX. ... or perms.XXX
```

Comment:
If these config settings are in use in a lot of projects, would it make sense to have that as fallback? 

Something like:
```
env('ROLES_GUI_CREATE_ROLE_MIDDLEWARE_TYPE', env('ROLES_GUI_CREATE_ROLE_MIDDLWARE_TYPE', 'admin')),
```